Rubberduck.RibbonDispatcher/RibbonCommonExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Banner, braces, whitespace - `(key??""` should be `(key ?? ""` - or better, `(key ?? string.Empty`.